cmd/minikube/cmd/delete.go

```diff
@@ -142,7 +142,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	if purge && len(profilesToDelete) > 1 && !deleteAll {
 		out.ErrT(style.Notice, "Multiple minikube profiles were found - ")
 		for _, p := range profilesToDelete {
-			out.Step(style.Notice, "    - {{.profile}}", out.V{"profile": p.Name})
+			out.Step(style.Notice, out.NoSpinner, "    - {{.profile}}", out.V{"profile": p.Name})
```

Comment:
Rather than adding an extra argument: the styles are designed to carry all information necessary for display. I would  suggest adding a `Spin` bool to `style.Options`:

https://github.com/kubernetes/minikube/blob/ead11c65cd233e45ac824e6ec8d77cbc27bbf806/pkg/minikube/style/style.go#L37

Reply:
Great idea! thanks @tstromberg from 42 files we reduce it to 6